stable/concourse/templates/NOTES.txt

```diff
@@ -33,19 +33,8 @@
     kubectl port-forward --namespace {{ .Release.Namespace }} $POD_NAME 8080:{{ .Values.concourse.atcPort }}
     {{- end }}
   {{- end }}
-{{- if .Values.concourse.basicAuth.enabled }}
-
-* Login with the following credentials
-  {{ if .Values.secrets.create }}
-  Username: {{ .Values.secrets.basicAuthUsername }}
-  Password: {{ .Values.secrets.basicAuthPassword }}
-  {{ else }}
-  Username: see basic-auth-username in secret {{ template "concourse.concourse.fullname" . }}
-  Password: see basic-auth-password in secret {{ template "concourse.concourse.fullname" . }}
-  {{ end }}
-{{- end }}
 * If this is your first time using Concourse, follow the tutorial at https://concourse-ci.org/hello-world.html
-{{- if contains "naive" .Values.concourse.baggageclaimDriver }}
```

Comment:
Many other helm charts have a section in their notes that describe how to log in, it could be written similar to https://github.com/helm/charts/blob/7ff0c6038c398488eb6ca8e7a1643af3f22c0452/stable/phabricator/templates/NOTES.txt#L50, but if you're deferring that til later, a subsequent PR can fix the docs.

Reply:
For local auth section, the reason we removed it is, as shown in values.yaml `secrets.localUsers`, the password is bcrypted so even one could get the local users USERNAME:PASSWORD string from `kubectrl get secret`, there is no way to decrypt the PASSWORD. So no sure what to put here. I agree that a further PR might be better.